sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserFacadeEjb.java

```diff
@@ -129,6 +131,47 @@
 			.collect(Collectors.toList());
 	}
 
+	@Override
+	/*
+	 * Get all users with the next higher jurisdiction, whose location contains the current users location
+	 * For facility users, this includes district and community users, if their district/community is identical with that of the facility
+	 */
+	public List<UserReferenceDto> getUsersWithSuperiorJurisdiction(UserDto user) {
+		JurisdictionLevel superordinateJurisdiction =
+			JurisdictionHelper.getSuperordinateJurisdiction(UserRole.getJurisdictionLevel(user.getUserRoles()));
+
+		List<User> superiorUsersList;
+
+		switch (superordinateJurisdiction) {
+		case NATION:
+			return null;
```

Comment:
The list actually has to contain all national-level users in this case - otherwise this code breaks the change concerning allowing region-level users to assign tasks to nation-level users

Reply:
Can we assume there is always only one nation per SORMAS instance? So We do not need a method named `getAllByNationAndUserRoles`

Reply:
Right now that's the case, yes